contracts/lbp/LBPManagerFactory.sol

```diff
@@ -0,0 +1,141 @@
+/*
+
+██████╗░██████╗░██╗███╗░░░███╗███████╗██████╗░░█████╗░░█████╗░
+██╔══██╗██╔══██╗██║████╗░████║██╔════╝██╔══██╗██╔══██╗██╔══██╗
+██████╔╝██████╔╝██║██╔████╔██║█████╗░░██║░░██║███████║██║░░██║
+██╔═══╝░██╔══██╗██║██║╚██╔╝██║██╔══╝░░██║░░██║██╔══██║██║░░██║
+██║░░░░░██║░░██║██║██║░╚═╝░██║███████╗██████╔╝██║░░██║╚█████╔╝
+╚═╝░░░░░╚═╝░░╚═╝╚═╝╚═╝░░░░░╚═╝╚══════╝╚═════╝░╚═╝░░╚═╝░╚════╝░
+
+*/
+
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+// solium-disable linebreak-style
+pragma solidity ^0.8.6;
+
+import "../utils/CloneFactory.sol";
+import "openzeppelin-solidity/contracts/access/Ownable.sol";
+import "./LBPManager.sol";
+
+/**
+ * @title PrimeDAO LBPManager Factory
+ * @dev   Enable PrimeDAO governance to create new LBPManager contracts.
+ */
+contract LBPManagerFactory is CloneFactory, Ownable {
+    address public lbpManagerMasterCopy;
```

Comment:
why do we call this variable `lbpManagerMasterCopy` do we have any other master copies?

Reply:
If I understand it correctly then you mean why not call it just `masterCopy`?

Reply:
yes

Reply:
that's correct :)